.github/workflows/build.yml

```diff
@@ -31,6 +32,7 @@ jobs:
       - name: Check for source code changes
         id: changes
         run: |
+          git diff origin/master HEAD --name-status -- . ':!openfpga_flow' ':!docs'
```

Comment:
This looks redundant with the next line.

Reply:
I added this to log in console, what change we see while running `change_detect`.